<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QTextEdit" name="textEdit_upload"/>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox_model">
      <item>
       <property name="text">
        <string>Select model</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>UNETR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SWIN-UNETR</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_upload">
      <property name="text">
       <string>Upload Files</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" alignment="Qt::AlignTop">
     <widget class="QPushButton" name="pushButton_run">
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_download">
      <property name="text">
       <string>Download Files</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QTextEdit" name="textEdit_run"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>997</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
